src/pudl/metadata/sources.py

```diff
@@ -487,7 +487,9 @@
         ),
         "field_namespace": "ferc714",
         "working_partitions": {
-            "years": sorted(set(range(2021, 2022))),  # Pre-XBRL data is monolithic
+            # 2021 and later data is in XBRL and not yet supported.
+            # 2006-2020 data is in monolithic CSV files, so any year means all years.
+            "years": sorted(set(range(2006, 2021))),
```

Comment:
The problem is it's used in multiple places for multiple purposes. Both to specify the partitions that can be pulled out of the datapackage, and to display the temporal coverage of the dataset.  Maybe a warning is a good idea. I guess it could replace the existing warning about the data being "interim".

Reply:
Also, also, we're trying to get to a place where only folks doing development / data engineering work are actually running the ETL.

Reply:
PUnting this for now.